{{ cookiecutter.project_slug }}/.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8, 3.9, "3.10"]
+        python-version: [3.9, "3.10", 3.11]
```

Comment:
am I correct in thinking this doesn't need quotation marks because it's not a trailing zero?